test/integration/consensus/consensus_sunny_day.cpp

```diff
@@ -42,7 +42,8 @@ class FixedCryptoProvider : public MockYacCryptoProvider {
     // TODO 15.04.2018 x3medima17 IR-1189: move to separate class
     auto size =
         shared_model::crypto::DefaultCryptoAlgorithmType::generateKeypair()
-            .publicKey().size();
+            .publicKey()
+            .size();
```

Comment:
This change looks redundant. Is it intentional?

Reply:
clang-format

Reply:
So, the issue is resolved.